src/raven.js

```diff
@@ -1174,16 +1174,22 @@ Raven.prototype = {
             }
 
             var fetchInput = args[0];
+
             var method = 'GET';
             var url;
 
             if (typeof fetchInput === 'string') {
               url = fetchInput;
-            } else {
+            } else if (
+              typeof 'Request' !== undefined &&
```

Comment:
This will always return `true`, as `typeof 'Request'` is always `string`.
Use `'Request' in _window` to verify that `Request` is available.